add snapshot consideration for model governance (#8004)
fishtown-analytics/docs.getdbt.com: website/snippets/_model-governance-rollback.md

#### Considerations

There are some considerations to keep in mind when using model governance features:

> - Model governance features like model access, contracts, and versions strengthen trust and stability in your dbt project. Because they add structure, it can make it harder to roll changes back later (for example, removing model access) and increases maintenance if adopted too early.

Reviewer: ```suggestion
- Model governance features like model access, contracts, and versions strengthen trust and stability in your dbt project. Because they add structure, they can make rollbacks harder (for example, removing model access) and increase maintenance if adopted too early.
```

>   Before adding governance features, consider whether your dbt project is ready to benefit from them. Introducing them too soon can make future changes harder if your models are still changing/evolving.

Reviewer: ```suggestion
  Before adding governance features, consider whether your dbt project is ready to benefit from them.  Introducing governance while models are still changing can complicate future changes.
```
